locales/vi/validation-inline.php

```diff
@@ -35,16 +35,16 @@
     'date'                 => 'Trường này không phải là định dạng của ngày-tháng.',
     'date_equals'          => 'Trường này phải là một ngày bằng với :date.',
     'date_format'          => 'Trường này không giống với định dạng :format.',
-    'declined'             => 'This value must be declined.',
-    'declined_if'          => 'This value must be declined when :other is :value.',
+    'declined'             => 'Giá trị này phải bị từ chối..',
```

Comment:
```suggestion
    'declined'             => 'Giá trị này phải bị từ chối.',
```
Extra full stop at the end of a sentence.